exercise_150508_Random_Menu_Generator.rb

```diff
@@ -0,0 +1,42 @@
+#### FILE_TITLE: exercise_150508_Random_Menu_Generator.rb
+
+#### Establish initial arrays
+vegetables = ["brussel sprout", "celery", "broccoli", "red cabbage", "pumpkin", "kale", "spinach", "wheatgrass", "avocado", "bell pepper", "cucumber", "squash", "tomato", "zucchini", "cauliflower", "pea", "peanut", "snap pea", "asparagus", "garlic", "lemongrass", "leek", "onion", "carrot", "ginger", "potato", "rutabaga", "sweet potato", "turnip", "yam", "wakame (seaweed)", "kombu (seaweed)"]
+proteins = ["chicken", "turkey", "duck", "goose", "salmon", "tilapia", "tofu", "pork", "pulled pork", "ground beef", "steak", "turtle", "lentil", "greek yogurt", "cottage cheese", "egg", "tuna", "sardines", "black bean", "kidney bean", "bacon", "peanut butter"]
+grains = ["quinoa", "barley", "wild rice", "oats", "millet", "noodles", "brown rice", "white rice", "tortillas", "potatoes", "sandwich", "crackers", "popcorn", "spaghetti"]
+
+#### Remove duplicates in the arrays
+vegetables.uniq!
+proteins.uniq!
+grains.uniq!
+
+#### Shuffle arrays; Find minimum array length
+veg_length = vegetables.shuffle!.length
```

Comment:
I might only use the `shuffle!` here and move the `length` into the `max_options` declaration since you don't need to use the local variables that store the length anywhere else.